src/components/graph/Graph.js

```diff
@@ -4,12 +4,13 @@ import Motivation from "./motivation/Motivation";
 import Mindset from "./mindset/Mindset";
 import Archetype from "./archetype/Archetype";
 import Filters from "./Filters";
-import { withFileContext } from "../../context/FileContext";
+import { useFileContext } from "../../context/FileContext";
 import "./Graph.scss";
 import DownloadResult from "./download/DownloadResult";
 
-const Graph = ({ fileData: data, history }) => {
+function Graph({ history }) {
```

Comment:
Why switchhing to function instead of arrow ?

Reply:
Because fast refresh (the replacement of hit module replacement for React) works nice only with function, not with unlined one. On function, it will be able to keep state whereas classes or inline functions will completely be reloaded. 
One other great thing is that you have a displayName instead of arrow functions which don’t.